Project-Aurora/Project-Aurora/Profiles/Dota 2/Layers/Dota2AbilityLayerHandler.cs

```diff
@@ -71,8 +71,11 @@ public override EffectLayer Render(IGameState state)
                     for (int index = 0; index < dota2state.Abilities.Count; index++)
                     {
                         Ability ability = dota2state.Abilities[index];
-                        if (ability.Name.Contains("seasonal"))
+                        if (ability.Name.Contains("seasonal") || ability.Name.Contains("high_five"))
+                        {
+                            index++;
```

Comment:
This is a double increment, i guess that wasn't intended?

Reply:
Correct, i shoudn't code late at night :D.

Shouldn't matter anyway because the abilties we want to skip are at the end anyway.

Thanks, ill change that.